test/parallel/test-process-versions.js

```diff
@@ -69,3 +76,9 @@ for (let i = 0; i < expected_keys.length; i++) {
 
 assert.strictEqual(process.config.variables.napi_build_version,
                    process.versions.napi);
+
+const expectedUndiciVersion = undici.version;
+assert.strictEqual(process.versions.undici, expectedUndiciVersion);
```

Comment:
Looks like you’d want to regenerate the files

Reply:
Didn't get you? 😅

Reply:
Oh are you talking about updating the versions since i guess undici  has been upgraded to 5.13

Reply:
Done!